bety_brapi/swagger/swagger.yaml

```diff
@@ -119,6 +119,100 @@ tags:
     description: "Docs"
     url: "https://wiki.brapi.org/"
 paths:
+  /getTreatmentsByExperiment:
```

Comment:
we can't add new endpoints to brapi, this is an official spec.

Reply:
ok. how should i use these new methods then? just remove the api parts?

Reply:
removed